linux/debian/8/foxpass_setup.py

```diff
@@ -39,19 +39,31 @@ def main():
     parser.add_argument('--bind-user', required=True, help='Bind User')
     parser.add_argument('--bind-pw', required=True, help='Bind Password')
     parser.add_argument('--api-key', required=True, help='API Key')
-    parser.add_argument('--ldap-uri', default='ldaps://ldap.foxpass.com', help='LDAP Server')
-    parser.add_argument('--api-url', default='https://api.foxpass.com', help='API Url')
+    parser.add_argument('--ldap-uri', '--ldap', default='ldaps://ldap.foxpass.com', help='LDAP Server')
+    parser.add_argument('--secondary-ldap', dest='ldaps', default=[], action='append', help='Secondary LDAP Server(s)')
+    parser.add_argument('--api-url', '--api', default='https://api.foxpass.com', help='API Url')
+    parser.add_argument('--secondary-api', dest='apis', default=[], action='append', help='Secondary API Server(s)')
     parser.add_argument('--ldap-connections', default=2, help='Number of connections to make to LDAP server.')
 
     args = parser.parse_args()
 
     binddn = 'cn=%s,%s' % (args.bind_user, args.base_dn)
 
+    uris = ['uri %s' % args.ldap_uri]
```

Comment:
style: send in the array of uris to write_nslcd_conf, and let it handle the formatting (adding the uri prefix).